Scripts/Mac/Tracing/Docs/cddl1.txt

```diff
@@ -0,0 +1,385 @@
+
```

Comment:
I'm not sure about licensing.  How did we arrive at this license?

Reply:
Note the commit message - https://github.com/Microsoft/VFSForGit/pull/737/commits/a5176525b4b0a84e686a8df5a49ddb4137fbc861

`dtruss` itself originates in the DTrace toolkit scripts in Solaris, at the time owned by Sun, which in turn was later bought by Oracle. Dtrace and the various toolkit scripts including `dtruss` were ported to what was then known as Mac OS X by Apple around 2006-2007, and included in the Mac OS X 10.5 "Leopard" release in 2007. Sun released DTrace along with most of their Open Source code under the CDDL.

The version I'm working from is the one from Apple's dtrace-285.200.15 release, located at https://opensource.apple.com/source/dtrace/dtrace-284.200.15/DTTk/dtruss.auto.html and also included in recent versions of macOS at `/usr/bin/dtruss`.

I included the license text in its original location of Docs/cddl1.txt as that's also the location referenced in the dtruss source comments.